src/library_async.js

```diff
@@ -205,6 +205,11 @@ mergeInto(LibraryManager.library, {
       var id = {{{ makeGetValue('ptr', C_STRUCTS.asyncify_data_s.rewind_id, 'i32') }}};
       var name = Asyncify.callStackIdToName[id];
       var func = Module['asm'][name];
+#if RELOCATABLE
```

Comment:
Please add a comment here, as it is not obvious why this is needed (I actually don't know right now).

Reply:
They also get added to asmLibraryArg which acts as a kind of global namespace

Reply:
@sbc100 So should we read from `asmLibraryArg` here, perhaps?

Reply:
Hmm.. I'm not really sure.  Perhaps you we want to use the `resolveGlobalSymbol` helper from library_dylink.js... although I'm sure what happens in the face of minification of imports/exports.. I guess that can't/doesn't happen with dynamic linking.

Reply:
In that case, I think this PR is good to land? If we think of a better solution here we can do that later, but for now this seems good enough. And I don't have any other comments, unless you did @sbc100 ?